src/main/java/com/exactpro/th2/readcsv/Main.java

```diff
@@ -156,13 +156,16 @@ private static Collection<RawMessage.Builder> attachHeaderOrHold(
     ) {
         String sessionAlias = streamId.getSessionAlias();
         HeaderInfo headerForAlias = headerHolder.getHeaderForAlias(sessionAlias);
+        if (LOGGER.isTraceEnabled()) {
+            LOGGER.trace("attachHeaderOrHold builders: " + builders.stream().map(MessageUtils::toJson).collect(Collectors.joining("\n")));
```

Comment:
Let's not use `\n` in logs. It will make the formatting awful (especially in grafana)